charts/v3.1/cray-hms-bss/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bss-env-config
+data:
+  addvertisAddress: {{ .Values.addvertisAddress }}
```

Comment:
Is this supposed to be advertise? 

I don't think this needs to be in a configMap if it is an environment variable that is being provided as part of the deployment. I had mentioned a configMap in the slack thread, but Kevin had suggested just an environment variable that was configured via customizations.

Reply:
Its probably supposed to be `advertiseAddress`. This is something that has been in the BSS chart for a long time, and is also baked into other places like customizations.yaml: https://github.com/Cray-HPE/csm/blob/main/vendor/stash.us.cray.com/scm/shasta-cfg/stable/customizations.yaml#L759-L760 


The configmap route path makes it possible to sanely override the environment variable when using the cray-service chart. The cray-service chart specifies environment variables as an array of elements. The issue with helm is that you can patch a single element of the array, you have to specify all of the elements if you override the environment variable array.

Reply:
As Ryan noted, this probably should be  `advertiseAddress`. However, this has been spelled this was since the cloud-init stuff was added to BSS by Ryan Bezdicek. The reason this looks new is because of the way we store multiple versions of our helm charts, and so copying to create the new ones makes it look like a new change. Trying to change that at this time is beyond the scope of this change, would require additional testing, etc.